isacc_messaging/api/views.py

```diff
@@ -175,3 +175,17 @@ def execute_requests():
         f"Execution succeeded for CommunicationRequest resources:\n{success_list}",
         f"Execution failed for CommunicationRequest resources:\n{error_list}"
     ])
+
+
+@base_blueprint.cli.command("maintenance-update-next-outgoing")
+@click.option("--simulate", is_flag=True, default=False, help="Simulate execution; don't persist to FHIR store")
```

Comment:
tiny nitpick: slightly prefer dry-run for consistency
```suggestion
@click.option("--dry-run", is_flag=True, default=False, help="Simulate execution; don't persist to FHIR store", simulate)
```